plugin.service.gto/addon.xml

```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.12" provider-name="Birger Jesch">
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.23" provider-name="Birger Jesch">
     <requires>
-        <import addon="xbmc.python" version="2.24.0"/>
+        <import addon="xbmc.python" version="2.25.0"/>
```

Comment:
Without reviewing further, xbmc.python = 2.25.0 is for Krypton.
Your PR is against the Jarvis branch.

Reply:
I use indeed some new JSON-RPC introduced with Krypton (PVR capabiltities). So I don't know what is the best way to bump the addon to Krypton, so excuse me.
 - should I discard/reject the PR here and mark the addon as broken/deprecated?
- Make a new PR on Krypton?
- Or can you suggest me another way to continue development only on krypton (and maybe later versions). What is the usual procedure here?

I use PyCharm as my preferred IDE which has some Github capabilities.

Reply:
This is a good question.

If you want to only support a Krypton version and force your current users to upgrade:

1) Please note that you shouldn't/can't submit a PR with xbmc.python set to a value that corresponds to a Kodi version higher than the branch you're submitting to. 

2) Submit a PR on the Jarvis branch setting the addon.xml field \<broken\>\</broken\>. You can include a message between to let them know there's a Krypton version available, e.g. \<broken\>Please upgrade to Krypton to continue using this add-on\</broken\>. You can actually delete all files except addon.xml, if you wish.

3) Then submit a PR on the Krypton branch with xbmc.python set to 2.25.0 for the new (higher) version of the add-on that you want to support.

Reply:
Thank you very much for your explanation. I'll do the necessary steps at monday, my dev system is @work.